pages/index.js

```diff
@@ -78,42 +77,39 @@ export default function Home() {
                 </h2>
-                <div className="text-base-darkest line-height-sans-6">
-                  PHDI&apos;s current body of work also falls within the scope
-                  of CDC&apos;s&nbsp;
-                  <span className="font-semibold text-underline text-primary link">
-                    <a
-                      className="underline text-primary"
-                      href="https://www.cdc.gov/surveillance/projects/dmi-initiative/index.html"
-                      target='_blank'
-                      rel="noreferrer">
-                      Data Modernization Initiative (DMI)
-                    </a>
-                  </span>
-                  ,&nbsp;which is an effort to modernize core public health data
-                  and surveillance infrastructure across the federal and state
-                  public health landscape.
+                <div className="text-base-darkest line-height-sans-6 font-public-sans line-height-sans">
+                  <p className="font-semibold">
+                    Automate your public health data processing so you can focus on taking meaningful public
+                    health action
+                  </p>
                   <br />
+                  <p>
+                    Our interdisciplinary team of researchers and engineers works to solve your jurisdiction&apos;s data challenges
+                    and integrate our flexible, modern tools into your existing workflows
+                  </p>
                   <br />
-                  <span className="desktop:padding-right-5">
-                    PHDI&apos;s work focuses on building and providing access to
-                    open source, modular software and tooling, known as
-                    “Building Blocks,” that states, territories, localities,
-                    tribes (STLTs), and other public health actors can integrate
-                    into their current workflows to reduce manual processes and
-                    solve challenges when working with public health data. Some
-                    Building Blocks offer relatively simple functionality, like
-                    standardizing patient names, while others perform more
-                    complex tasks, including geocoding and standardizing
-                    addresses.
-                  </span>
+                  <p className="font-semibold">
+                    Improve the quality and completeness of your data
+                  </p>
                   <br />
+                  <p>
+                    Using a pipeline of modular software components known as Building Blocks, we validate,
+                    clean, standardize, and enrich your data — all while reducing the need for manual processes
+                  </p>
+                  <br />
+                  <p className="font-semibold">
+                    Get the data that you need in the format that you want it
+                  </p>
+                  <br />
```

Comment:
We should not be using `<br/>` to create spaces between paragraphs. We should use CSS, something like the one below, or create a new CSS class.
 ```
  p {
    margin-bottom: 16px; 
  }
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
Agreed with Josh here, the problem is <br/> and screen readers don't play well so its an accessibility issue. Looks like it may have been like that before this PR but it should be an easy tweak to fix here? Other than that this PR looks great tho!